chapters/equations.tex

```diff
@@ -762,6 +762,15 @@ \section{Initialization, initial equation, and initial algorithm}\label{initiali
 \lstinline!when initial() then! or \lstinline!when {$\ldots$, initial(), $\ldots$} then!. In this case, the algorithmic statements within the when-statement remain active during the whole
 initialization phase.
 
+An active when-clause inactivates the following \lstinline!elsewhen! (similarly as for when-clauses during simulation), but apart from that
+the first \lstinline!elsewhen initial() then! or \lstinline!elsewhen {$\ldots$, initial(), $\ldots$} then! is similarly active during initialization as
+\lstinline!when initial() then! or \lstinline!when {$\ldots$, initial(), $\ldots$} then!.
+
+\begin{nonnormative}
+That means that any subsequent \lstinline!elsewhen initial()! has no effect,
+similarly as \lstinline!when false then!.
```

Comment:
Could we go as far as to speak of warnings?
```suggestion
A tool may warn in case an \lstinline!initial()! condition cannot activate its when-clause due to an earlier when-clause already being active during initialization.
```

Reply:
I considered it, but there are (as indicated) other similar cases as well, and to me it looks odd if we discuss warnings for the obscure cases - and not for the simple ones; and thought it sufficed to say that it has no effect.